src/routes/klageskjema/summary/text-formatting.tsx

```diff
@@ -6,3 +6,5 @@ export const foedselsnrFormat = (fnr: string): string => {
     }
     return fnr;
 };
+
+export const navnFormat = (navn: string): string => navn.toLowerCase().replace(/(^|\s)\S/g, l => l.toUpperCase());
```

Comment:
Kunne vi erstattet denne funksjonen med å sette navn i ALL CAPS, slik vi gjør ellers?

Da blir det likt som Dekoratøren og resten av applikasjonen vår. Samt at vi slipper alle potensielle edge-caser med navn som ikke skal formateres på samme måte.

Reply:
https://nav-it.slack.com/archives/G01GKNG797E/p1611048806018600